### ISSUE TITLE
[Reborn] Mac: Low FPS in game.

### ISSUE THREAD

**Issue by Vuccappella** [ValveSoftware/Dota-2#1784]
I'm experiencing very low frame rates inside of Reborn.
http://i.imgur.com/lPJ5JeE.jpg 15-20fps. My video settings are the same as on the main client where I easily get between 115 and 120fps.

There are slight frame drops while navigating the menu as well but it's not too noticeable compared to ingame gameplay.

Configuration: http://i.imgur.com/DJgP8tu.png

**Comment by lucasmorano** [comment]
Just to reinforce that this is exactly happenning to me also. There' some serious leakage going..

**Comment by bklooster** [comment]
Yup, the performance of the OS X reborn client is virtually unplayable. I cranked it down to low, just in order to play through a co-op game I joined. (I guess I was optimistic about the improvements) I barely made it through it. 

Looking at CPU profiler it is a vacuum of cpu, running at 550-600% cpu on my quad-core. The input is incredibly laggy. The hardware mouse seems to be updating, but the actual input in the game seems significantly delayed. If I had to wager a guess, I'd say that you're hitting an extension that OpenGL is having to emulate on the CPU somewhere and this is causing a huge CPU drain. 

I also can't see the heroes on the hero profile screen (the large 3 hero view). 

I can't close the console after I open it. 

My computer specifications: https://gist.github.com/bklooster/6026046
